release.config.js

```diff
@@ -1,5 +1,5 @@
 module.exports = {
-    branch: 'release',
+    branches: ['release', { name: 'master', prerelease: 'beta' }],
```

Comment:
We should probably keep the `'+([1-9])?(.{+([1-9]),x}).x'` regex to allow [hotfixes for older versions](https://github.com/semantic-release/semantic-release/blob/beta/docs/recipes/maintenance-releases.md#releasing-a-bug-fix-for-version-10x-users).